src/__stories__/tag-story.tsx

```diff
@@ -42,61 +42,63 @@ export const Default: StoryComponent<Args> = ({label: labelFromArgs}) => {
     const getLabel = (fallback: string) => labelFromArgs || fallback;
 
     return (
-        <Inline space={48} dataAttributes={{testid: 'tags'}}>
-            <Container>
-                <Text3 medium>Example</Text3>
-                <Tag Icon={IconOfferPercentFilled} type="promo" dataAttributes={{qsysid: 'promo'}}>
-                    {getLabel('Promo')}
-                </Tag>
-                <Tag Icon={IconStarFilled} type="active">
-                    {getLabel('Active')}
-                </Tag>
-                <Tag Icon={IconTimeFilled} type="inactive">
-                    {getLabel('Inactive')}
-                </Tag>
-                <Tag Icon={IconCheckRegular} type="success">
-                    {getLabel('Success')}
-                </Tag>
-                <Tag Icon={IconAlertRegular} type="warning">
-                    {getLabel('Warning')}
-                </Tag>
-                <Tag Icon={IconCloseRegular} type="error">
-                    {getLabel('Error')}
-                </Tag>
-            </Container>
+        <div data-testid="tags" style={{display: 'inline-flex'}}>
```

Comment:
Added this div to make the screenshot pass. Otherwise, puppeteer takes a screenshot including the `<Inline/>` negative margins